dot/peerset/peerstate.go

```diff
@@ -261,12 +263,15 @@ func (ps *PeersState) addNoSlotNode(idx int, peerID peer.ID) {
 
 func (ps *PeersState) removeNoSlotNode(idx int, peerID peer.ID) {
 	if _, ok := ps.sets[idx].noSlotNodes[peerID]; !ok {
+		logger.Debugf("peer %s already does not exist in no slot node", peerID)
```

Comment:
Why `already`? :thinking:

Reply:
we are trying to remove something, but it is already not there

Reply:
Do you mean it already left?
As in, `already does not exist` doesn't add up to me, it could `already exist` but not the other way around if you see what I mean? But that's just me supernitting though

Reply:
```suggestion
		logger.Debugf("peer %s is not in "no slot node" map", peerID)
```

what about something like this?